.buildkite/npm-build.sh

```diff
@@ -12,11 +12,11 @@ npm run build
 
 echo "--- Test"
 
-npx tsdx test --collect-coverage 
+npm run test -- --collect-coverage
 
-echo "--- Lint"
+#echo "--- Lint"
```

Comment:
The lint refactor is deferred until all open PRs are merged.

Reply:
What is the lint refactor? Why are the lint rules changing? I took the defaults from the tsdx template and they seem fine to me.

Reply:
Changing to use [standard-js](https://standardjs.com/), which is what `cardano-graphql` already uses, and has been around a lot longer that prettier.

Reply:
Maybe you should change `cardano-graphql` to use [prettier](https://prettier.io/) then. Isn't newer supposed to be better in nodejs world? :stuck_out_tongue_winking_eye:

Rather than commenting out the lint step, why not add `|| true` to the command, so we can at least see what it's suggesting.

Now you've changed it, how do I run the linter anyway? The old command `tsdx lint` just crashes now. This should be added to the readme.

Reply:
> Isn't newer supposed to be better in nodejs world?

Ha, well not in this case.
> Rather than commenting out the lint step, why not add || true to the command, so we can at least see what it's suggesting.

Yer could have done that, but I didn't want it to kill the job so just disabled it. Run it locally if you're curious

> The old command tsdx lint just crashes now. This should be added to the readme.

It was always abstracted by the `npm run lint`, and was never included in the Readme. I don't really see it as critical since the `package.json` is self-documenting.